supervisor/api/ingress.py

```diff
@@ -78,6 +78,19 @@ async def create_session(self, request: web.Request) -> Dict[str, Any]:
         session = self.sys_ingress.create_session()
         return {ATTR_SESSION: session}
 
+    @api_process
+    async def validate_session(self, request: web.Request) -> Dict[str, Any]:
+        """Create a new session."""
+        self._check_ha_access(request)
+
+        # Check Ingress Session
+        session = request.cookies.get(COOKIE_INGRESS)
+        if not self.sys_ingress.validate_session(session):
+            _LOGGER.warning("No valid ingress session %s", session)
+            raise HTTPUnauthorized()
```

Comment:
This needs to be excepted in `api_process` so we can return it like all other API calls

Reply:
Are you sure? It's already there for the other ingress endpoints.

Also realized that I do it now based on cookies, rewriting it to JSON data like other APIs.

Reply:
For the other calls that raise those, they are not using `api_process`

Reply:
In `security.py`, the security middleware also just raises this error. It's a security incident, not an error.

Reply:
I don't know how do you want to handle this on the frontend. If you want like /ingress/session then yes, You need a response with an APIError().